frontend/packages/text-editor/src/TextList.tsx

```diff
@@ -49,6 +49,7 @@ export const TextList = ({
               textRowEntries={row.translations}
               variables={row.variables || []}
               selectedLanguages={selectedLanguages}
+              showButton={row.textKey !== 'appName'}
```

Comment:
Nice! One more thing you could do is to move `'appName'` to the constants.js file and import it here. Not important, just a thought I had 😊

Reply:
moved